frontend/pages/journey/[id].vue

```diff
@@ -348,9 +549,13 @@ const handleUpload = (result: string) => {
                 <SvgDashboardIcon class="h-6 w-6" />
                 <p class="text-2xl hover:underline">Dashboard</p>
             </NuxtLink>
-            <SvgMenu
-                class="mx-5 h-8 w-8 hover:cursor-pointer md:mx-10 md:h-10 md:w-10"
-                @click="visibleSidebar = true"
+            <span
```

Comment:
HAndyversion:
Remove the Word "Dashboard" and put the Dashboard sign to the left upper corner.